tests/test_integration/test_substitution.py

```diff
@@ -119,7 +120,7 @@ def test_substitution_env(repo, dev_template, pipeline_id, suffix):
     create_tag(repo, "1.2.3")
     create_file(repo)
 
-    env = {"ANOTHER_ENV": "345"}
+    env = {"ANOTHER_ENV": "345", "PATH": os.getenv("PATH", ""), "PYTHONPATH": os.getenv("PYTHONPATH", "")}
```

Comment:
This test checks that environment variables can be properly substituted. Passing PATH or PYTHONPATH should not change test result

Reply:
Thank you for reviewing! The reason it makes a difference in Nix is that PYTHONPATH is used to communicate to the Python process where all of the package dependencies are (Nix builds each Python package into a separate directory). When the test starts its separate Python process, it cannot find the coverage or setuptools modules in Nix unless PYTHONPATH is passed to it. I found also that PATH needs to be passed as well (maybe to retain access in that process to git).

Reply:
I mean all other tests does not pass neither PATH nor PYTHONPATH to child process. Why patching just this test instead of function which created the process in the first place?

Reply:
Patching that function is a good idea. Let me do that!

Reply:
Gave it a try